js/mapModel.js

```diff
@@ -7,7 +7,7 @@ class mapModel{
     }
 
     addTile(Ter, X, Y, Visit, Item){
-        if(this.mapHash.getTile(X, Y) == undefined) {
+       if(this.mapHash[X+','+Y] === undefined) {
             this.mapHash[X + ',' + Y] = new Tile(Ter, X, Y, Visit, Item);
         }else{
             this.mapHash.modTile(Ter, X, Y, Visit, Item);
```

Comment:
This location also has the incorrect usage of `this.mapHash.ModTile`. Should be `this.ModTile`

Reply:
Good eye...made the necessary change